scraper/index.js

```diff
@@ -27,11 +27,11 @@ function getStockDetails(stockId) {
 
       ratios = { stockId: stockId };
       const getMarketCap = () => parser.parse(rawRatios[0][1].trim());
-      const getPe = () => parser.parse(rawRatios[3][0].trim());
+      // const getPe = () => parser.parse(rawRatios[3][0].trim());
```

Comment:
I would say let's leave this PE here. We can refresh this at a later point in time via some other endpoint.

Reply:
I have fixed the PE problem. Do you still want the change?